tests/storages_tests/test_storages.py

```diff
@@ -862,32 +860,23 @@ def check_set_and_get(trial_id: int, key: str, value: Any) -> None:
 
 
 @pytest.mark.parametrize("storage_mode", STORAGE_MODES)
-@pytest.mark.parametrize("include_best_trial", [True, False])
-def test_get_all_study_summaries(storage_mode: str, include_best_trial: bool) -> None:
+def test_get_all_studies(storage_mode: str) -> None:
 
     with StorageSupplier(storage_mode) as storage:
-        expected_summaries, _ = _setup_studies(storage, n_study=10, n_trial=10, seed=46)
-        summaries = storage.get_all_study_summaries(include_best_trial=include_best_trial)
-        assert len(summaries) == len(expected_summaries)
-        for _, expected_summary in expected_summaries.items():
-            summary: Optional[StudySummary] = None
-            for s in summaries:
-                if s.study_name == expected_summary.study_name:
-                    summary = s
+        expected_frozen_studies, _ = _setup_studies(storage, n_study=10, n_trial=10, seed=46)
+        frozen_studies = storage.get_all_studies()
+        assert len(frozen_studies) == len(expected_frozen_studies)
+        for _, expected_frozen_study in expected_frozen_studies.items():
+            frozen_study: Optional[FrozenStudy] = None
+            for s in frozen_studies:
+                if s.study_name == expected_frozen_study.study_name:
+                    frozen_study = s
                     break
-            assert summary is not None
-            assert summary.direction == expected_summary.direction
-            assert summary.datetime_start == expected_summary.datetime_start
-            assert summary.study_name == expected_summary.study_name
-            assert summary.n_trials == expected_summary.n_trials
-            assert summary.user_attrs == expected_summary.user_attrs
-            assert summary.system_attrs == expected_summary.system_attrs
-            if include_best_trial:
-                if expected_summary.best_trial is not None:
-                    assert summary.best_trial is not None
-                    assert summary.best_trial == expected_summary.best_trial
-            else:
-                assert summary.best_trial is None
+            assert frozen_study is not None
+            assert frozen_study.direction == expected_frozen_study.direction
```

Comment:
[Notes] This is out-of-scope of this PR, but this test case does not cover `FrozenStudy.directions`.